src/main/java/com/atlassian/bitbucket/jenkins/internal/client/BitbucketCommitClientImpl.java

```diff
@@ -22,20 +26,40 @@ public BitbucketCommitClientImpl(BitbucketRequestExecutor bitbucketRequestExecut
 
     @Override
     public BitbucketCommit getCommit(String identifier) {
```

Comment:
I would annotate this as `@Nullable`, or have it return `Optional<BitbucketCommit>` (not too fussed which)